ui/react-components/components/AddAppointment/AddAppointment.jsx

```diff
@@ -22,20 +23,27 @@ import {
     getRecurringAppointmentsConflicts,
     saveAppointment,
     saveRecurring
-} from "./AddAppointmentService";
+} from "../../services/AppointmentsService/AppointmentsService";
 import Label from '../Label/Label.jsx';
 import {getDateTime, isStartTimeBeforeEndTime} from '../../utils/DateUtil.js'
 import TimeSelector from "../TimeSelector/TimeSelector.jsx";
 import AppointmentNotes from "../AppointmentNotes/AppointmentNotes.jsx";
-import RecurringPlan from "../RecurringPlan/RecurringPlan.jsx";
+import AppointmentPlan from "../AppointmentPlan/AppointmentPlan.jsx";
```

Comment:
It makes the code much more complicated to understand, detail comments shared on commits